macrosynergy/management/utils/sparse.py

```diff
@@ -90,7 +90,8 @@ def create_delta_data(
 
     if not isinstance(df, QuantamentalDataFrame):
         raise ValueError("`df` must be a QuantamentalDataFrame")
-
+    if not isinstance(return_density_stats, bool):
```

Comment:
@Magnus167, did we accidentally merge in changes from the `sparse.py` script in another PR? I guess once we merge in the other changes to dev it should be fixed? Or is it a matter of merging in the develop branch?

Reply:
This is from merging dev back in I believe